### PR TITLE
Creating a file

### PR DESCRIPTION
**Pull request by FlyingMedusa**
### Description:

Script accepts url and prints an ordered list

### CODE REVIEW

File: Internet data/fetch_links.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+def main():
+    url = input("Please paste the url\t")
+
+    ols_children = extract_ol(url)
+
+    for li_list in ols_children:
+        print()
+        for nr, li in enumerate(li_list, 1):
```
Comment (gaperlinski) [inline Internet data/fetch_links.py:24]: Impressing use of `enumerate`

File: Internet data/fetch_links.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    return children
+
+
+def main():
+    url = input("Please paste the url\t")
+
+    ols_children = extract_ol(url)
+
+    for li_list in ols_children:
+        print()
```
Comment (gaperlinski) [inline Internet data/fetch_links.py:23]: why do you need an empty `print()` statement here?

File: Internet data/fetch_links.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# https://flyingmedusa.github.io/simple_page/index.html
+
+
+def extract_ol(url):
+    page = urllib.request.urlopen(url)
+    html = BeautifulSoup(page.read(), 'html.parser')
+    ol_nodes = html.find_all('ol')
+    children = []
+    for ol in ol_nodes:
+        children.append(ol.findChildren())
```
Comment (gaperlinski) [inline Internet data/fetch_links.py:13]: checking the documentation? nice

**Review by gaperlinski — COMMENTED** [review]
Well done

**Comment by gaperlinski** [comment]
Go ahead and merge this pull request when you feel like it

**Review by gaperlinski — APPROVED** [review]

File: Internet data/fetch_links.py (hunk 2 above)
Comment (FlyingMedusa) [reply Internet data/fetch_links.py:23]: Just for the esthetics. When a page has more than one ordered list, this empty print() separates lists. I believe the output is more user-friendly now.

**Pull request merged**